models/org_team.go

```diff
@@ -158,6 +162,24 @@ func (t *Team) addRepository(e Engine, repo *Repository) (err error) {
 	return nil
 }
 
+// addAllRepositories adds all repositories to the team.
+// If the team already has some repositories they will be left unchanged.
+func (t *Team) addAllRepositories(e Engine) error {
+	err := e.Iterate(&Repository{OwnerID: t.OrgID}, func(i int, bean interface{}) error {
```

Comment:
Why not just change includesAllRepository to true?

Reply:
All repositories need to be added to the team because access rights management won't work without them.
It is the same behavior as for owners teams.

Reply:
OK. And I think we should use a better method to batch insert a slice of `team_repo`.

Reply:
Sure we can use a better method to batch insert a slice of `team_repo` but the method `addRepository` do a lot more than that.
I don't think the batch insert will be worth it compared to the whole processing.

Reply:
Assume there are 1000 repositories on this organization. And the code here is not work with sqlite database.

Reply:
OK. This is my last issue I think it should be resolved before merge.

Reply:
Here is a rough record of time spend in the different blocks of the method `addRepository` for 1000 repositories and 100 members:
- addTeamRepo + update team: 580ms (0%)
- recalculateTeamAccesses: 57s (45%)
- watchRepo: 1m10s (55%)

As I said before, there's no gain to batch insert a slice of `team_repo`
However there's one easy update: to not reload members for each repository.

What does not work with sqlite database?

Reply:
sqlite database only allow one connection, you insert data on an iterate, that will return 500.

Reply:
Thanks for the clarification.
I have no such error because the method is called with a session and uses only one connection.

Reply:
It still return 500 on my local test.

Reply:
I can't reproduce the issue. I must do it wrong.
I build with `TAGS="bindata sqlite sqlite_unlock_notify" make clean generate build`
I tested on Linux and Windows.
Do the UT pass on your local machine?
Can you please give me the steps which lead to the error 500?

Reply:
That's what I compiled locally. I'm on macOS. Just grant team to all organization repositories.

Reply:
The UT must fail, don't they?

Reply:
Problematic in pgSQL as well (500). After some debugging I found hasTeamRepo get this error: https://github.com/lib/pq/issues/635